config/types.go

```diff
@@ -33,13 +33,19 @@ const (
 	// TaggedAddressWAN is the map key for WAN tagged addresses.
 	TaggedAddressWAN = "wan"
 
+	// Name of the dataplane's container
+	ConsulDataplaneContainerName = "consul-dataplane"
+
 	// Match Consul: https://github.com/hashicorp/consul/blob/68e79b8180ca89e8cfca291b40a30d943039bd49/agent/consul/authmethod/awsauth/aws.go#L16-L20
 	AuthMethodType         string = "aws-iam"
 	IAMServerIDHeaderName  string = "X-Consul-IAM-ServerID"
 	GetEntityMethodHeader  string = "X-Consul-IAM-GetEntity-Method"
 	GetEntityURLHeader     string = "X-Consul-IAM-GetEntity-URL"
 	GetEntityHeadersHeader string = "X-Consul-IAM-GetEntity-Headers"
 	GetEntityBodyHeader    string = "X-Consul-IAM-GetEntity-Body"
+
+	SyntheticNode    string = "synthetic-node"
+	ECSSyntheticNode string = "ecs-synthetic-node"
```

Comment:
My thought behind introducing this field is to help the UI show a ECS specific message similar to k8s. May be the banner now needs to show a generic image rather than a k8s specific one. I can remove it for now